transport_nantes/asso_tn/tests_og_twitter.py

```diff
@@ -0,0 +1,208 @@
+from django.test import TestCase, Client
+from topicblog.models import (TopicBlogItem, TopicBlogTemplate,
+                              TopicBlogContentType)
+from django.contrib.auth.models import User
+from datetime import datetime, timezone
+
+
+class TestOg_Twitter(TestCase):
+    def setUp(self):
+        # Creates a user for FKs
+        self.user = User.objects.create(username='user-og-twitter',
+                                        password='mdp-og-twitter')
+        # Creates a base content type for FKs
+        self.content_type = TopicBlogContentType.objects.create(
```

Comment:
There are two types of pages.  Pages, like TopicBlog, source their images from MEDIA.  Pages like legal source their images from STATIC.  The base template has to respond to this distinction, checking for media or static.

You are only testing one case here.

Reply:
Ok I look the legal source and the jobs page seems to be STATIC too but they have no og or twitter data on the template of job i can see  {% block og_title %} but there no  {% block og_title %} on the base template. I will try too look more deep for that and implement the test for this kind of page.